habitat-hitl/habitat_hitl/environment/gui_pick_helper.py

```diff
@@ -10,6 +10,12 @@
 import numpy as np
 
 DIST_HIGHLIGHT: Final[float] = 0.15
+COLOR_GRASPABLE: Final[mn.Color3] = mn.Color3(1, 0.75, 0)
```

Comment:
This class was not being used at all in `main`. I made some major changes to get it to work for `rearrange_v2`.